ios/FluentUI/Notification/MSFNotificationView.swift

```diff
@@ -149,8 +170,6 @@ import UIKit
         }
     }
 
-    private var anchorView: UIView?
-    private var animated: Bool = true
     private var completionsForHide: [() -> Void] = []
     private var constraintWhenHidden: NSLayoutConstraint!
     private var constraintWhenShown: NSLayoutConstraint!
```

Comment:
isHiding is a bit below this, but we used to also have a public isShown. Did we want to add that back? I think we can just use the UIView's isHidden.

Reply:
Agreed, I don't think isShown and isHidden are both needed anymore, one should do.